nilearn/datasets/atlas.py

```diff
@@ -750,8 +781,10 @@ def fetch_coords_power_2011():
     fdescr = _get_dataset_descr(dataset_name)
     package_directory = os.path.dirname(os.path.abspath(__file__))
     csv = os.path.join(package_directory, "data", "power_2011.csv")
-    params = dict(rois=np.recfromcsv(csv), description=fdescr)
-
+    params = dict(rois=pd.read_csv(csv), description=fdescr)
+    if legacy_format:
+        warnings.warn(LEGACY_FORMAT_MSG)
+        params['rois'] = params['rois'].to_records()
```

Comment:
Here also I think the pandas index gets added WRT to the result returned on the main branch